simulation/sub8_gazebo/worlds/contact.world

```diff
@@ -84,14 +91,18 @@
           </geometry>
         </collision>
       </link>
-    </model>
+    </model> -->
+    <include>
```

Comment:
That's what I get for not being able to run Gazebo on my hometop

Jacob Panikulam | (630) - 877 - 7665
Software Lead, 2016 Subjugator Team <http://subjugator.org/>
Github <https://github.com/jpanikulam> | About Me
<http://jpanikulam.github.io/>

On Sat, May 7, 2016 at 10:23 AM, mattlangford <notifications@github.com>
wrote:

> In simulation/sub8_gazebo/worlds/contact.world
> <https://github.com/uf-mil/Sub8/pull/126#discussion_r62416928>:
>
> > @@ -84,14 +91,18 @@
> >            </geometry>
> >          </collision>
> >        </link>
> > -    </model>
> > +    </model> -->
> > +    <include>
>
> I had only used the ground plane instead of the transdec to test my
> mission, did you want to have that be in master?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/uf-mil/Sub8/pull/126/files/70803231d6847ed5bff27658560f6b4a10c0749e#r62416928>
>